<ui version="4.0" >
 <class>IGarminExport</class>
 <widget class="QDialog" name="IGarminExport" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Create gmapsupp.img</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Output Path:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QLabel" name="labelPath" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolPath" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>File Prefix</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="linePrefix" />
     </item>
     <item row="1" column="2" >
      <widget class="QPushButton" name="pushExport" >
       <property name="text" >
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QPushButton" name="pushClose" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushClose</sender>
   <signal>clicked()</signal>
   <receiver>IGarminExport</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>808</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel" >
     <x>427</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
